chart/templates/jobs/cache-monitor/job.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 The HuggingFace Authors.
+
+{{- if .Values.images.jobs.cacheMonitor }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels: {{ include "labels.cacheMonitor" . | nindent 4 }}
+  name: "{{ include "name" . }}-job-cache-monitor"
+  namespace: {{ .Release.Namespace }}
+spec:
+  ttlSecondsAfterFinished: 300
```

Comment:
we can let it live longer (one day?) after it finished, in order to be able to look at the logs

Reply:
Sure